opm/input/eclipse/Schedule/Group/GroupKeywordHandlers.cpp

```diff
@@ -268,7 +293,7 @@ void handleGCONPROD(HandlerContext& handlerContext)
                 // GCONPROD
                 // 'G1' 'ORAT' 1000 100 200 300 RATE =>  constraints 100,200,300 should be honored
                 if (production.cmode == Group::ProductionCMode::ORAT ||
-                    (groupLimitAction.allRates != Group::ExceedAction::NONE &&
+                    (groupLimitAction.oil != Group::ExceedAction::NONE &&
                     !apply_default_oil_target)) {
                     production.production_controls |= static_cast<int>(Group::ProductionCMode::ORAT);
```

Comment:
`production_controls` are actually constraints or limits. control will be constraint with an action `RATE`.  It is some naming thing to do.